index.html

```diff
@@ -294,7 +294,26 @@ <h4>Computation steps</h4>
                 </pre>
               </details></div>
             </li>
-            <li id="step2C">Otherwise, if computing a <a class="termref" data-lt="accessible name">name</a>, and if the <code>current node</code> has an <code>aria-label</code> <a class="termref">attribute</a> whose value is not the empty string, nor, when trimmed of white space, is not the empty string:
+            <li id="step2C">Otherwise, if the <code>current node</code> is a control embedded within the label (e.g. the <code>label</code> element in HTML or any element directly referenced by <code>aria-labelledby</code>) for another <a class="termref">widget</a>, where the user can adjust the embedded control's value, then include the embedded control as part of the text alternative in the following manner:
```

Comment:
This may be something to address separately, but I worry that it's confusing that we say "include the embedded control as part of the text alternative" rather than "return" like we do everywhere else. Although we say "otherwise" in later steps, "include" here could be misconstrued to suggest that we should proceed with the following steps even if this step 2C applied.

Reply:
Tracked with issue https://github.com/w3c/accname/issues/126